### PR TITLE
Bump Symfony 6 to PHP 8

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 6.0
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | -
| License       | MIT
| Doc PR        | -

:tada:

### CODE REVIEW

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @nicolas-grekas.

File: .travis.yml [hunk 1]
```diff
@@ -14,16 +14,16 @@ addons:
 
 env:
     global:
-        - SYMFONY_VERSION=5.x
-        - MIN_PHP=7.2.5
+        - SYMFONY_VERSION=5.4
```
Comment (derrabus) [inline .travis.yml:17]: `6.0`?

Comment (nicolas-grekas) [reply .travis.yml:17]: indeed, and similar in 5.3 & 5.4

**Comment by hacfi** [comment]
Just a note: there is an issue with PHP 8.0.5 and Postgresql which is why 8.0.6 was released.

**Comment by nicolas-grekas** [comment]
For the record, 8.0.2 was chosen as the minimum because that's the version that runs on Travis.

**Comment by derrabus** [comment]
8.0.2 is a reasonable requirement. We can still fine tune this before November, but a Postgres compat issue is not enough to block 8.0.5 imho.

**Comment by hacfi** [comment]
I'm also fine with 8.0.2. Just wanted to note this in case someone is setting up a project with 8.0.5 and then gets errors (happened to me recently).

**Comment by stof** [comment]
Well, in any case, the *recommended* version is always the uptodate patch release, to benefit from PHP bugfixes. This constraint is about the minimum version, not the recommended one.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -216,8 +216,6 @@ jobs:
 
       - name: Install dependencies
         run: |
-          echo "::group::fake PHP version"
-          composer config platform.php 8.0.99
```
Comment (derrabus) [inline .github/workflows/tests.yml:220]: We still need this, actually.

Comment (derrabus) [reply .github/workflows/tests.yml:220]: #41347

**Pull request merged**